packages/contracts-bedrock/contracts/deployment/SystemDictator.sol

```diff
@@ -160,6 +161,15 @@ contract SystemDictator is OwnableUpgradeable {
      *         initialized upon deployment.
      */
     constructor() {
+        SystemConfig.ResourceConfig memory rcfg = SystemConfig.ResourceConfig({
```

Comment:
Should these be zero values, to avoid giving the impression that they are actually impactful?

Reply:
Yes I agree, altho we do not do that for all of our contracts, would prefer to place null values in the impls constructor's call to `initialize` but would enforce that all contracts are tested behind a proxy which tbh kind of a good thing

Reply:
I agree with you but we cannot use all 0 values because of certain invariants that must hold with regards to the config values. Slight preference to leave as the default values so we know that the config will not cause a revert, basically the reverts exist to prevent dummy configs from being passed in.